### ISSUE TITLE
Crashes Playnite at startup

### ISSUE THREAD

**Issue by OneNot** [Lacro59/playnite-howlongtobeat-plugin#204]
The HowlongToBeat addon crashes Playnite on app start. (v3.2.1)
It doesn't seem to happen after the first restart required to enable the addon, but if you just keep restarting the app it starts happening, and after it happens once, it happens every time as long as the app is launched with the extension enabled.
I made sure to disable all other addons and themes and can still reproduce the crash.

I wiped the extensions.log and reproduced the crash, so I could get a short and sweet log for you, but it doesn't really show anything wrong that I can tell.
[extensions.log](https://github.com/Lacro59/playnite-howlongtobeat-plugin/files/14157101/extensions.log)

**Comment by OneNot** [comment]
Okay just read through issue #200 and yeah, looks like this is the same issue.
Here's what I now did:
1. Wiped the logs to get nice and easy to read logs of only the stuff happening starting from this point
2. Installed HLTB addon
3. Restarted from the UI popup -> Worked fine
4. Restarted manually x3 -> worked fine each time
5. Authenticated
6. Restarted manually x2 _> crashed each time

Here are the logs:
[extensions.log](https://github.com/Lacro59/playnite-howlongtobeat-plugin/files/14157227/extensions.log)
[playnite.log](https://github.com/Lacro59/playnite-howlongtobeat-plugin/files/14157228/playnite.log)

**Comment by Verssgn** [comment]
I had the same issue, I removed it and installed 3.1.2 (3.2.0 and 3.2.1 keep crashing for me) and it doesn't crash anymore.

**Comment by OneNot** [comment]
> I had the same issue, I removed it and installed 3.1.2 (3.2.0 and 3.2.1 keep crashing for me) and it doesn't crash anymore.

Even when authenticated?

**Comment by Verssgn** [comment]
When I installed the older versions the config stayed meaning I was authenticated in all of the 3 versions. 

There is a chance I didn't uninstall the addon, and I just installed the older versions which overwrote it, cant remember, but I am currently on 3.1.2 and it doesn't crash anymore, and the addon is authenticated.

**Comment by iWockstar** [comment]
It was causing a crash for me too after not having an issue for a week. I'm assuming it's an internal issue with the plugin.

**Comment by wensleyoliv** [comment]
I'm having this crash too.

https://github.com/Lacro59/playnite-howlongtobeat-plugin/assets/14226926/4fa4191c-d16b-4982-9ab7-1b8bae8aab07

**Comment by Jeffs24** [comment]
I have exactly the same problem. I was starting to remove my extensions one by one, hopelessly tying to find the source of the crashes, until I stumbled on [this reddit thread](https://www.reddit.com/r/playnite/comments/194e37j/updated_how_long_to_beat_causes_startup_failure/).

The crashes stropped completely when I disabled this extension. I'm really not sure why it started now. I looked through the playnite.log and extensions.log, and there is absolutely nothing about the crashes or this plugin.
